holidata/holidays/hr-HR.py

```diff
@@ -7,18 +7,19 @@
 class hr_HR(Locale):
     """
     01-01: [NF] Nova Godina
-    01-06: [NRF] Bogojavljanje / Sveta tri kralja
+    01-06: [NRF] Bogojavljanje ili Sveta tri kralja
```

Comment:
I guess the name of the holiday is not "Bogojavljanje ili Sveta tri kralja" ("Epiphany or Holy Three Kings") but either "Bogojavljanje" _or_ "Sveta tri kralja" - which depends on ... circumstances...? Which name is more ("Bogojavljanje" / "Sveta tri kralja") common?

This might be a general problem for holidata to tackle...

Reply:
> I guess the name of the holiday is not "Bogojavljanje ili Sveta tri kralja" ("Epiphany or Holy Three Kings") but either "Bogojavljanje" _or_ "Sveta tri kralja" - which depends on ... circumstances...? Which name is more ("Bogojavljanje" / "Sveta tri kralja") common?
> 
> This might be a general problem for holidata to tackle...

All I can say is that in the official docs it is written as: "Bogojavljenje ili Sveta tri kralja".

Reply:
I am going to pick "Bogojavljenje" as it used in the already existing holidata